make-macosx-app.sh

```diff
@@ -372,6 +372,8 @@ PLIST="${PLIST}
     <string>NSApplication</string>
     <key>NSHighResolutionCapable</key>
     <false/>
+	<key>NSRequiresAquaSystemAppearance</key>
+	<false/>
```

Comment:
Minor nitpick; this uses tabs while the rest of the generated Info.plist uses spaces. If your editor supports it, enabling displaying whitespace makes it easier to match existing formatting.

Reply:
Good catch, I'll fix it.